### PR TITLE
fixed some style issues

### PR DESCRIPTION
**Pull request by a-m-thomas**
In progress. Need to add xrefs and expand on ACID support, then review again for readability for new users.

### CODE REVIEW

**Review by lennessyy — APPROVED** [review]

File: modules/intro/pages/continuous-availability-overview.adoc [hunk 1]
```diff
@@ -7,14 +7,14 @@ HA (High Availability) is a generic term used to describe a computer system whic
 With Continuous Availability, TigerGraph goes beyond the standard scope and definition of High Availability by providing the following functionality:
 
 * Fault tolerance against loss of database server(s)
- ** Automated recovery of services for Intra-cluster failure
+ ** Automated recovery of services in case of intra-cluster failure
 * Full native HA support for user-facing applications - Seamless automatic client reconnection to standby GSQL server and GraphStudio servers
-* Failover to remote cluster during Disaster Recovery
-* Improve RoI with additional Replicas
- ** Enhanced Query Throughput performance
+* Failover to remote cluster during disaster decovery
```
Comment (victor-gsl) [inline modules/intro/pages/continuous-availability-overview.adoc:12]: 1. typo: decovery -> recovery
2. This comparison chart is not just informational, but it is Promotional material, where terms are key terms are highlighted. In this case, they are industry-standard terms help the reader to see what features are supported. I think you should have left it the way it was.

File: modules/intro/pages/index.adoc [hunk 2]
```diff
@@ -46,17 +46,18 @@ For a more detailed description of the TigerGraph release process, please see th
 
 == Architecture overview
 The TigerGraph Platform runs on standard, commodity-grade Linux servers. 
-The core components (GSE and GPE) are implemented in C++ for optimal performance. TigerGraph system is designed to fit into your existing environment with a minimum of fuss.
+The core components of the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE are implemented in C++ for optimal performance. TigerGraph is designed to fit into your existing environment with a minimum of fuss.
```
Comment (victor-gsl) [inline modules/intro/pages/index.adoc:49]: 1. If you want to use commas, then you need one more to fully bracket the explanatory phrase "or GPE",
changing
_the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE are_
to
_the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE, are_

2. I personally prefer to use parentheses when introducing an acronym. I feel it is more streamlined.
_the Graph Storage Engine (GSE) and the Graph Processing Engine (GPE) are_

File: modules/intro/pages/multigraph-overview.adoc [hunk 3]
```diff
@@ -8,10 +8,10 @@ MultiGraph enables several powerful use cases:
...
-* Overlapping graphs: Graphs can partially overlap, to enable a combination of shared and private data.
-* Hierarchical subgraphs: A Graph X can be defined to cover the domains of Graphs Y and Z, that is, Graph X = (Graph Y) U (Graph Z). This provides an interesting way to describe a data partitioning or parent-child structure. (This is not the same as defining sub-classes of data types; data types are still independent.)
+* Overlapping graphs: Graphs can partially overlap to enable a combination of shared and private data.
+* Hierarchical subgraphs: A graph X can be defined to cover the domains of Graphs Y and Z, that is, Graph X = (Graph Y) U (Graph Z). This provides an interesting way to describe a data partitioning or parent-child structure. (This is not the same as defining sub-classes of data types; data types are still independent.)
```
Comment (victor-gsl) [inline modules/intro/pages/multigraph-overview.adoc:12]: Inconsistency: If `graph` is capitalized in `Graph Y`, then `Graph X` should be capitalized.
Note that in my equation (I think I am the original author), the variables are `Graph X`, `Graph Y`, and `Graph Z`, not `X`, `Y`, and `Z`. You could rewrite it.

File: modules/intro/pages/transaction-and-acid.adoc [hunk 4]
```diff
@@ -43,12 +47,19 @@ Multiple reads in a single read-only transaction T1 will get the same results, e
 [discrete]
 ==== *Durability*  +++<a id="TigerGraphTransactionProcessingandACIDPropertiesSupport-Durability">++++++</a>+++
 
-Committed transactions are written to disk (SSD or HDD). The TigerGraph platform implements write-ahead logging (WAL) to provide durability.
+Committed transactions are written to disk. The TigerGraph platform implements write-ahead logging (WAL) to provide durability.
```
Comment (victor-gsl) [inline modules/intro/pages/transaction-and-acid.adoc:50]: I think you should leave the parenthetical phrase. It answers implied questions about what sort of disks does TigerGraph support, in a concise and unobtrusive way (in my opinion).

**Pull request merged**